kernel/packages/decentraland-ecs/src/ecs/Attachable.ts

```diff
@@ -0,0 +1,17 @@
+import { IEntity, IEngine } from "./IEntity"
+
+/**
+ * Entities can be attached to each other by using the `setParent` method. However, there are cases where we might want to attach entities
+ * to other objects that are not entities created by the same scene (for example, the player's avatar). For those cases, we have this class.
+ */
+export abstract class Attachable {
```

Comment:
Why use an abstract class and not an Interface?

Is it to be able to use `Attachable.AVATAR`?

Reply:
No, I can only use it like `Attachable.AVATAR` with abstract classes

Reply:
I was trying to confirm it, but I clearly didn't phrase it correctly haha